biztalk/technical-guides/using-the-performance-analysis-of-logs-pal-tool.md

```diff
@@ -44,70 +44,70 @@ The PAL (Performance Analysis of Logs) tool reads in a performance monitor count
+|         BizTalk Server:  BizTalk Messaging Latency:  Request Response Latency         |                                                                                                                                                                                                                                                                                                                                                              Average latency in milliseconds from when the Messaging Engine receives a request document from the adapter until the time a response document is given back to the adapter. Refer to the chart showing how latency is measured in BizTalk Inbound Latency Analysis in this topic. Assuming a low latency environment, this analysis checks for a Request-Response Latency greater than 5 seconds. This may indicate a processing delay between the inbound adapter and the outbound adapter.<br /><br /> References:<br /><br /> -  [Request/Response Messaging](https://go.microsoft.com/fwlink/?LinkId=155288)<br />-   [BizTalk Server 2006: Scalability Case Study Using the SOAP Adapter in BizTalk Server 2006](https://go.microsoft.com/fwlink/?linkid=108774)                                                                                                                                                                                                                                                                                                                                                              |
```

Comment:
```suggestion
|         BizTalk Server:  BizTalk Messaging Latency:  Request Response Latency         |                                                                                                                                                                                                                                                                                                                                                              Average latency in milliseconds from when the Messaging Engine receives a request document from the adapter until the time a response document is given back to the adapter. Refer to the chart showing how latency is measured in BizTalk Inbound Latency Analysis in this topic. Assuming a low latency environment, this analysis checks for a Request-Response Latency greater than 5 seconds. This may indicate a processing delay between the inbound adapter and the outbound adapter. <br /><br /> References: <br /><br />- [Request/Response Messaging](/biztalk/core/request-response-messaging) <br />- [Scaling Your Solutions](/biztalk/core/scaling-your-solutions)                                                                                                                                                                                                                                                                                                                                                              |
```